pkgs/development/libraries/gdal/default.nix

```diff
@@ -271,6 +267,9 @@ stdenv.mkDerivation (finalAttrs: {
     # flakey tests, to remove on next release
```

Comment:
Can these be removed?

Reply:
`test_vsiaz_fake_write` was marked as flaky in https://github.com/OSGeo/gdal/commit/b21dcc52aaecd5edaf510ea70bfeb05d622648cf#diff-ff89ab96d4c6a2de306624bc0b7ad8dc66e0d4720e229fe58bf340751e55bc06R460 . 

`test_vsiaz_write_blockblob_chunk_size_1` doesn't look to be present in GDAL 3.9.3. 

Both can be dropped.

Reply:
Done.